samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -302,49 +306,50 @@ public void testSample() throws Exception {
     assertThat(out).contains("19 Venue 19 1200100");
     assertThat(out).contains("42 Venue 42 390650.99");
 
+    out = runSample("querywithjson");
```

Comment:
This should probably also be removed, if we remove the duplicate sample in the monolith sample file.